ee/app/livechat-enterprise/server/lib/LivechatEnterprise.js

```diff
@@ -163,6 +165,20 @@ export const LivechatEnterprise = {
 		updateInquiryQueuePriority(roomId, priority);
 		updateRoomPriorityHistory(roomId, user, priority);
 	},
+
+	placeRoomOnHold(room) {
+		const { _id: roomId } = room;
+		if (!roomId) {
+			return;
+		}
+		LivechatRooms.setOnHold(roomId);
+		Subscriptions.setOnHold(roomId);
+		LivechatRooms.unsetCanPlaceOnHold(roomId);
```

Comment:
This should not be a method as described above.

Reply:
Removed this property as its no longer required